### ISSUE TITLE
Campaign page fails WCAG 2.1 -- Contast issues

### ISSUE THREAD

**Issue by bsouster** [wet-boew/GCWeb#1643]
[Page in question] (https://wet-boew.github.io/themes-dist/GCWeb/campaign2-en.html)

The buttons over the banner fail non-text contrast no matter which colour you compare luminosity (button colour #F5F5F5, border #979797, page colour #FFF). 

Also the button text (#3368A6) fails against the button hover and down states (#CFD1D5, #BBBFC5).

![image](https://user-images.githubusercontent.com/1944744/75638263-02ca0600-5bf9-11ea-910b-e1f673069f62.png)

Success criteria in question:
[Non-text contast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
[Contrast (minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)

**Comment by EricDunsworth** [comment]
FWIW the button's link text colour is #335075 in the CSS. That produces a > 4.5:1 contrast ratio when measured against both the light grey (#f5f5f5) and dark grey (#cfd1d5) backgrounds.

@bsouster When you measured the button link text's colour as #3368A6, that was probably the result of anti-aliasing on the pixel you selected in your tool. I don't think that would technically count as a 1.3.4 failure given that the user agent was responsible for producing the anti-aliasing.

Based on my experience, portions of text characters that have been anti-aliased (like curved lines) tend to be slightly brighter than the "real" font colour. The real font colour can usually be picked up by measuring portions that haven't been anti-aliased (like flat horizontal/vertical lines).

I still think this issue is valid though since the button borders still fail 1.4.11.

**Comment by GormFrank** [comment]
@duboisp Do you think we should add styles for this?

**Comment by bsouster** [comment]
Hold on the button border change.

**Comment by bsouster** [comment]
Just wanted to verify: yes the borders need to pass since there is not other indication this is a button.

No, @EricDunsworth, I selected the correct area; screens factor into the colour when using pickers. For this reason I avoid passing items sitting on the line of 4.5:1 or 3:1.

![image](https://user-images.githubusercontent.com/1944744/81984818-60200180-9603-11ea-919d-3698fa8980eb.png)
